trade-finance-manager-api/src/v1/rest-mappings/mappings/gef-facilities/mapGefFacilityDates.js

```diff
@@ -30,11 +30,11 @@ const mapGefFacilityDates = (facility, facilityTfm, dealSnapshot) => {
     mapped.isUsingFacilityEndDate = isUsingFacilityEndDate;
 
     if (isUsingFacilityEndDate) {
-      mapped.facilityEndDate = convertDateToTimestamp(facilityEndDate);
+      mapped.facilityEndDate = facilityEndDate;
```

Comment:
@MarRobSoftwire  is it correct that tfm receives the facility end date as a `Date`? 
I'm unsure whether or not to keep it as a `Date` the whole way through, or to convert it to an ISO string the whole way through. 
Just realised I'm keeping it as a `Date` in the facility snapshot but I convert it to an ISO string in the amendments... what do you think is best?